models/include/cloe/component/actuator.hpp

```diff
@@ -24,6 +24,7 @@
 #include <boost/optional.hpp>  // for optional
 #include <cloe/component.hpp>  // for Component
 #include <fable/json.hpp>      // for Json
+#include <fable/utility/boost_optional.hpp>
```

Comment:
I would love to, but that would be a breaking change in the models, which is much more likely to affect downstream user code... and which I want to avoid for now.

Reply:
Hmm, actually, only an ABI change. Still, it's another PR to extricate boost from models package.